src/components/InputTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  state = {
+    title: '',
+  };
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.addTodoProps(this.state.title);
```

Comment:
Consider using destructuring for component props, as this is React best practice.